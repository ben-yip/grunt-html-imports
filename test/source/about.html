<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>About</title>
</head>
<body>

<link rel="import" href="common/_header.html">

<link rel="import" href="component/_menu.html">

<p>About page.</p>

<link rel="import" href="common/_footer.html">

</body>
</html>