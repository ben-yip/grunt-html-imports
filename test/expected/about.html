<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>About</title>
</head>
<body>

<header>
    This is header.
</header>

<p>Menu</p>
<ul>
    <li>menu item 1</li>
    <li>menu item 2</li>
    <li>menu item 3</li>
</ul>

<p>About page.</p>

<footer>
    This is footer.
</footer>

</body>
</html>